core/domain/skill_services.py

```diff
@@ -1014,7 +1047,10 @@ def compute_summary_of_skill(
     skill_model_worked_examples_count = len(
         skill.skill_contents.worked_examples)
 
-    skill_summary = skill_domain.SkillSummary(  # type: ignore[no-untyped-call]
+    # Ruling out the possibility of None for mypy type checking.
+    assert skill.created_on is not None
```

Comment:
Ditto, needs justification.

Reply:
`if skill.created_on is None` exception is added. Basically, we create a summary for those models that are fetched from the datastore and every model that is fetched from the datastore should have created_on and last_updated values because of the update_timestamps method.

Reply:
OK, thanks. But just to be clear, every time I say "needs justification", the comment should be in the code rather than in the PR review, so that other developers looking at the code in the future can see it too.

Reply:
I'm not sure if this was addressed? Just flagging.

Reply:
yeah this is addressed already